src/lib/extractor/extract.ts

```diff
@@ -168,6 +171,7 @@ export function fromProject(opts: Options): Doc.DocPackage {
   const diagnostics = project.getPreEmitDiagnostics()
   if (diagnostics.length) {
     const message = project.formatDiagnosticsWithColorAndContext(diagnostics)
+    // TODO It would be nice to have a --force to not throw on errors here
```

Comment:
Agreed!

`--skipTypeCheck` Might be more explicit. But I agree that a `-f/--force` is a conventional way to do this! So we should have `-f/--force`. We might discover that `--force` becomes multiple things, not just this, kind of like `strict` mode in TS. If so, it might make sense to eventually support both `--skipTypeCheck` and `--force`.